src/App.js

```diff
@@ -53,17 +55,87 @@ const App = () => {
     setResults(workout);
   };
 
+  const [removeExercises, setEx] = useState(null);
+  const removeEx = (workout) => {
+    setEx(workout);
+    setTotal((count) => count - 1);
+
+    const exerciseIndex = selectedExercises.findIndex(
+      (i) => i.name === workout.name
+    );
+    selectedExercises.splice(exerciseIndex, 1);
+
+    console.log("Exercise: " + workout.name);
+    console.log("Index: " + exerciseIndex);
+  };
+
+  const toggleDrawer = (anchor, open) => (event) => {
+    if (
+      event.type === "keydown" &&
+      (event.key === "Tab" || event.key === "Shift")
+    ) {
+      return;
+    }
+
+    setDrawer({ ...drawer, [anchor]: open });
+  };
+
+  const [drawer, setDrawer] = React.useState({
+    top: false,
+    left: false,
+    bottom: false,
+    right: false,
+  });
+
+  const [openSnackbar, setSnackbar] = React.useState(false);
+  const [snackbarName, setSnackbarName] = React.useState("");
+
+  const handleCloseSnackbar = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+
+    setSnackbar(false);
+  };
+
+  const selectCard = (workout, name, image) => {
+    const newArray = selectedExercises;
+    newArray.push({
+      name: name,
+      image: image,
+      cat: results.slug,
+      slug: workout.slug,
+    });
+
+    setSnackbar(true);
```

Comment:
We should maybe look at making one state for the snackbar instead of setting different properties of it. Something like 

const [snackbar, setSnackbar] = useState({name: 'example', open: false});

Just a suggestion, not necessary .

Reply:
I created a new Issue that focuses on code cleanup. I think this would be perfect for that! Since it works now, I don't want to mess it up for now. 

Can we add this as a todo in the Code Clean Up issue?

Reply:
Sweet!